### PR TITLE
Added genesis.yml file and BeSLab env scripts

### PR DESCRIPTION
**Pull request by asa1997**

### CODE REVIEW

File: beslab/0.0.1/besman-package-managers.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+function __besman_uninstall_package_managers()
+{
+    __besman_echo_yellow "Uninstalling npm gradle maven and pip"
+    sudo apt update
+    sudo apt purge --autoremove nodejs npm gradle maven python3-pip
+}
+
+# function __besman_install_npm()
```
Comment (anilsingla) [inline beslab/0.0.1/besman-package-managers.sh:17]: Are we going to use these functions ?

File: beslab/0.0.1/besman-private-artifact-repo.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+function __besman_install_jfrog()
+{
+    echo "Installing jfrog"
```
Comment (anilsingla) [inline beslab/0.0.1/besman-private-artifact-repo.sh:4]: Can we add Todo here? seems function is yet to develop right ?

File: beslab/0.0.1/besman-private-artifact-repo.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+}
+
+function __besman_uninstall_jfrog()
+{
+    echo "Uninstalling jfrog"
```
Comment (anilsingla) [inline beslab/0.0.1/besman-private-artifact-repo.sh:9]: Need a Todo comment here as well

File: beslab/0.0.1/besman-package-managers.sh (hunk 1 above)
Comment (asa1997) [reply beslab/0.0.1/besman-package-managers.sh]: Removed the functions. Thanks for pointing it out

File: beslab/0.0.1/besman-private-artifact-repo.sh (hunk 2 above)
Comment (asa1997) [reply beslab/0.0.1/besman-private-artifact-repo.sh:5]: Done

File: beslab/0.0.1/besman-private-artifact-repo.sh (hunk 3 above)
Comment (asa1997) [reply beslab/0.0.1/besman-private-artifact-repo.sh:11]: Done

**Review by anilsingla — APPROVED** [review]

**Pull request merged**